src/content/energy-consumption/index.md

```diff
@@ -4,65 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
-
-[CCRI](https://carbon-ratings.com) has examined the impact of Ethereum’s merge from proof-of-work to proof-of-stake; the results have underlined the significant impact of changing the consensus protocol: The annualized electricity consumption was reduced from 22,900,320 MWh to 2,601 MWh and thus by more than **99.988 %**. Likewise, Ethereum’s carbon footprint was decreased by approximately **99.992 %** (from 11,016,000 to 870 tonnes CO2e). Depicted metaphorically, this corresponds to a reduction in emissions from the height of the Eiffel Tower to a small plastic toy figure, as shown in the figure below. 
+From the very beginning, Ethereum aimed to implement a proof-of-stake based consensus mechanism, but doing so without sacrificing security and decentralization took years of focused research and development. Therefore, a proof-of-work mechanism was used to get the network started. Proof-of-work requires miners to use their computing hardware to calculate a value, expending energy in the process. Ethereum's total energy consumption peaked during the apex of the crypto bull market in February 2022 at just under 94 TWh/yr. Just before the switch to proof-of-stake, the energy consumption was closer to [78 TWh/yr](https://digiconomist.net/ethereum-energy-consumption), comparable to that of Uzbekistan, with a carbon emission equivalent to that of Azerbaijan (33 MT/yr). The switch to proof-of-stake solved this problem.
 
 ![Energy consumption comparison of pre- and post-Merge Ethereum. Displayed is on the left the Eiffel tower with 330 meters height and on the right a plastic toy figure with 4 cm height within a magnifying glass.](energy_consumption_pre_post_merge.png)
 
-Both proof-of-work and proof-of-stake are just mechanisms to decide who gets to add the next block. Swapping proof-of-work for proof-of-stake, where the real-world value invested comes from ETH staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.
-
-## Why proof-of-stake is greener than proof-of-work {#why-pos-is-greener-than-pow}
+[CCRI](https://carbon-ratings.com) examined the impact of Ethereum’s merge from proof-of-work to proof-of-stake. The annualized electricity consumption was reduced by more than **99.988 %**. Likewise, Ethereum’s carbon footprint was decreased by approximately **99.992 %** (from 11,016,000 to 870 tonnes CO2e). Depicted metaphorically, this corresponds to a reduction in emissions from the height of the Eiffel Tower to a small plastic toy figure, as shown in the figure above. Therefore, the environmental cost of securing the network is drastically reduced. Simultaneously, the security of the network is thought to have increased.
```

Comment:
> CCRI examined the impact of Ethereum’s merge from proof-of-work to proof-of-stake.

Weird phrasing. Some alternatives:

1. CCRI examined the impact of Ethereum’s transition from proof-of-work to proof-of-stake. 
2. CCRI examined the impact of The Merge (Ethereum's transition from proof-of-work to proof-of-stake.)
3. CCRI examined the impact of Ethereum’s Merge transition from proof-of-work to proof-of-stake.